action.yml

```
@@ -18,6 +18,10 @@ inputs:
     description: 'Additional options to pass to docker build command'
     default: ''
     required: false
+  build-path:
+    description: 'The build-path specifies where to find the files for the “context” of the build on the Docker daemon'

```

I would make it slightly shorter since it is already clear which variable we are setting.
```suggestion
    description: 'Location of the build context for the docker build command'

```

---

Done